packages/loader/container-loader/src/container.ts

```diff
@@ -340,6 +340,13 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
         return this._deltaManager.readonly;
     }
 
+    /**
+     * {@inheritDoc DeltaManager.storageOnly}
+     */
+    public get storageOnly() {
```

Comment:
I'd not expose this property at any level, unless there is very good reason for it.
If we need to expose it, I'd rather rework readonly & readonlyPermissions properties on container & deltamanager into sinngle property that returns enum of these values:
- state is unknown yet (did not connect to ordering service yet)
- not read-only
- read-only permissions
- read-only forced mode
- read-only due to looking at immutable version

Reply:
The app wants to be able to disambiguate this case form other readonly cases. i had considered the enum approach, but wasn't sure it was worth it, but should be too hard to add, and then deprecate the old properties